modules/cudf/src/data_frame.ts

```diff
@@ -192,6 +187,29 @@ export class DataFrame<T extends TypeMap = any> {
     return new DataFrame(series_map);
   }
 
+  /**
+   * Return a group-by on a single column.
```

Comment:
I see cuDF Python doesn't have a docstring for groupby, and instead has a whole [markdown file](https://github.com/rapidsai/cudf/blob/branch-0.19/docs/cudf/source/groupby.md) for it. Would it be good to add a little blurb about what `groupBy` is? Maybe there's a good description in the Postgres docs.

Reply:
I'm going to make an issue about docs today I will make expanding this one of the bullet points